### PR TITLE
[Localization] Localize prevent berry ability phase and translate

### PR DESCRIPTION
**Pull request by EnochG1**
## What are the changes?
Add the option to localize message text when PreventBerryUseAbAttr(Like Unnerve) prevents use berry.

## Why am I doing these changes?
All text should be localizable.

## What did change?
 - Change hard-coded text for localize > phases.ts
 - Added localization keys in each language folders. > ability-trigger.ts
 - Translated them to Korean.

### Screenshots/Videos
TBD (After translator's review)

#### En
image

#### Fr
image

#### De
image

#### Pt_BR
image

#### Zh_CN
image

#### Zh_TW
image

#### Ko
image

## How to test the changes?
Manually. (Using overrides.ts)
example) preventing 'Sitrus' case
```Typescript
  readonly ABILITY_OVERRIDE: Abilities = Abilities.UNNERVE;
  readonly MOVESET_OVERRIDE: Array<Moves> = [ Moves.FALSE_SWIPE ]; // Not to make opp pokemon faint.
  readonly OPP_HELD_ITEMS_OVERRIDE: Array<ModifierOverride> = [{name: "BERRY", type: BerryType.SITRUS}];
```

## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I considered writing automated tests for the issue?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Review by Tempo-anon — APPROVED** [review]
Gonna wait for this to be fully TL'd before merging

File: src/locales/fr/ability-trigger.ts [hunk 1]
```diff
@@ -59,4 +59,5 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postSummonSwordOfRuin": "L’Épée du Fléau de {{pokemonNameWithAffix}}\naffaiblit la {{statName}} des Pokémon alentour !",
   "postSummonTabletsOfRuin": "Le Bois du Fléau de {{pokemonNameWithAffix}}\naffaiblit l’{{statName}} des Pokémon alentour !",
   "postSummonBeadsOfRuin": "Les Perles du Fléau de {{pokemonNameWithAffix}}\naffaiblissent la {{statName}} des Pokémon alentour !",
+  "preventBerryUse": "{{pokemonNameWithAffix}} is too\nnervous to eat berries!",
```
Comment (Adri1) [inline src/locales/fr/ability-trigger.ts:62]:
```suggestion
  "preventBerryUse": "{{pokemonNameWithAffix}} est tendu\net ne peut plus manger de Baies !",
```

**Review by Adri1 — APPROVED** [review]

File: src/locales/de/ability-trigger.ts [hunk 2]
```diff
@@ -59,4 +59,5 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postSummonSwordOfRuin": "Unheilsschwert von {{pokemonNameWithAffix}} schwächt {{statName} aller Pokémon im Umkreis!",
   "postSummonTabletsOfRuin": "Unheilstafeln von {{pokemonNameWithAffix}} schwächt {{statName}} aller Pokémon im Umkreis!",
   "postSummonBeadsOfRuin": "Unheilsjuwelen von {{pokemonNameWithAffix}} schwächt {{statName}} aller Pokémon im Umkreis!",
+  "preventBerryUse": "{{pokemonNameWithAffix}} is too\nnervous to eat berries!",
```
Comment (CodeTappert) [inline src/locales/de/ability-trigger.ts:62]:
```suggestion
  "preventBerryUse": "{{pokemonNameWithAffix}} kriegt vor Anspannung keine Beeren mehr runter!!",
```

File: src/locales/zh_CN/ability-trigger.ts [hunk 3]
```diff
@@ -59,4 +59,5 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postSummonSwordOfRuin": "{{pokemonNameWithAffix}}的灾祸之剑\n令周围的宝可梦的{{statName}}减弱了！",
   "postSummonTabletsOfRuin": "{{pokemonNameWithAffix}}的灾祸之简\n令周围的宝可梦的{{statName}}减弱了！",
   "postSummonBeadsOfRuin": "{{pokemonNameWithAffix}}的灾祸之玉\n令周围的宝可梦的{{statName}}减弱了！",
+  "preventBerryUse": "{{pokemonNameWithAffix}} is too\nnervous to eat berries!",
```
Comment (40chyan) [inline src/locales/zh_CN/ability-trigger.ts:62]:
```suggestion
  "preventBerryUse": "{{pokemonNameWithAffix}}因太紧张\n而无法食用树果！",
```

File: src/locales/zh_TW/ability-trigger.ts [hunk 4]
```diff
@@ -59,4 +59,5 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postSummonSwordOfRuin": "{{pokemonNameWithAffix}}'s Sword of Ruin lowered the {{statName}}\nof all surrounding Pokémon!",
   "postSummonTabletsOfRuin": "{{pokemonNameWithAffix}}'s Tablets of Ruin lowered the {{statName}}\nof all surrounding Pokémon!",
   "postSummonBeadsOfRuin": "{{pokemonNameWithAffix}}'s Beads of Ruin lowered the {{statName}}\nof all surrounding Pokémon!",
+  "preventBerryUse": "{{pokemonNameWithAffix}} is too\nnervous to eat berries!",
```
Comment (40chyan) [inline src/locales/zh_TW/ability-trigger.ts:62]:
```suggestion
  "preventBerryUse": "{{pokemonNameWithAffix}}因太緊張\n而無法食用樹果！",
```

**Review by mercurius-00 — APPROVED** [review]

File: src/locales/pt_BR/ability-trigger.ts [hunk 5]
```diff
@@ -59,4 +59,5 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postSummonSwordOfRuin": "Sword of Ruin de {{pokemonNameWithAffix}} reduziu a {{statName}}\nde todos os Pokémon em volta!",
   "postSummonTabletsOfRuin": "Tablets of Ruin de {{pokemonNameWithAffix}} reduziu o {{statName}}\nde todos os Pokémon em volta!",
   "postSummonBeadsOfRuin": "Beads of Ruin de {{pokemonNameWithAffix}} reduziu a {{statName}}\nde todos os Pokémon em volta!",
+  "preventBerryUse": "{{pokemonNameWithAffix}} is too\nnervous to eat berries!",
```
Comment (josericardo-fo) [inline src/locales/pt_BR/ability-trigger.ts:62]:
```suggestion
  "preventBerryUse": "{{pokemonNameWithAffix}} está nervoso\ndemais para comer frutas!",
```

File: src/locales/de/ability-trigger.ts [hunk 6]
```diff
@@ -59,4 +59,5 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postSummonSwordOfRuin": "Unheilsschwert von {{pokemonNameWithAffix}} schwächt {{statName} aller Pokémon im Umkreis!",
   "postSummonTabletsOfRuin": "Unheilstafeln von {{pokemonNameWithAffix}} schwächt {{statName}} aller Pokémon im Umkreis!",
   "postSummonBeadsOfRuin": "Unheilsjuwelen von {{pokemonNameWithAffix}} schwächt {{statName}} aller Pokémon im Umkreis!",
+  "preventBerryUse": "{{pokemonNameWithAffix}} kriegt vor Anspannung keine Beeren mehr runter!!",
```
Comment (CodeTappert) [inline src/locales/de/ability-trigger.ts:62]:
```suggestion
  "preventBerryUse": "{{pokemonNameWithAffix}} kriegt vor Anspannung keine Beeren mehr runter!",
```

Sorry i did it wrong the first time. :(

File: src/locales/it/ability-trigger.ts [hunk 7]
```diff
@@ -59,4 +59,5 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postSummonSwordOfRuin": "{{pokemonNameWithAffix}}'s Sword of Ruin lowered the {{statName}}\nof all surrounding Pokémon!",
   "postSummonTabletsOfRuin": "{{pokemonNameWithAffix}}'s Tablets of Ruin lowered the {{statName}}\nof all surrounding Pokémon!",
   "postSummonBeadsOfRuin": "{{pokemonNameWithAffix}}'s Beads of Ruin lowered the {{statName}}\nof all surrounding Pokémon!",
+  "preventBerryUse": "{{pokemonNameWithAffix}} is too\nnervous to eat berries!",
```
Comment (NicusPulcis) [inline src/locales/it/ability-trigger.ts:62]:
```suggestion
  "preventBerryUse": "{{pokemonNameWithAffix}} non riesce a\nmangiare le bacche per l'agitazione!",
```

**Review by NicusPulcis — APPROVED** [review]
italian localization has been suggested

File: src/locales/es/ability-trigger.ts [hunk 8]
```diff
@@ -59,4 +59,5 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "postSummonSwordOfRuin": "{{pokemonNameWithAffix}}'s Sword of Ruin lowered the {{statName}}\nof all surrounding Pokémon!",
   "postSummonTabletsOfRuin": "{{pokemonNameWithAffix}}'s Tablets of Ruin lowered the {{statName}}\nof all surrounding Pokémon!",
   "postSummonBeadsOfRuin": "{{pokemonNameWithAffix}}'s Beads of Ruin lowered the {{statName}}\nof all surrounding Pokémon!",
+  "preventBerryUse": "{{pokemonNameWithAffix}} is too\nnervous to eat berries!",
```
Comment (EnochG1) [inline src/locales/es/ability-trigger.ts:62]:
```suggestion
  "preventBerryUse": "{{pokemonNameWithAffix}} está muy nervioso y no puede comer bayas!",
```

**Pull request merged**